packages/bot-skeleton/src/scratch/blocks/Math/math_number.js

```diff
@@ -6,6 +6,7 @@ Blockly.Blocks.math_number = {
 
         const fieldInput = this.getField('NUM');
         fieldInput.setValidator(input => this.numberValidator(input));
+        fieldInput.maxDisplayLength = 17;
```

Comment:
Blockly documentation for `maxDisplayLength` says `Maximum characters of text to display before adding an ellipsis.` So, will it limit the number of characters?

Reply:
No, the number of characters is not limited. The QA said that the user should be able to enter more characters, like in Binary bot, and i just have to limit the number that is displayed, so the input field wont grow too large.

Reply:
Got it 👍